modules/react/side-panel/lib/SidePanel.tsx

```diff
@@ -169,7 +168,7 @@ export class SidePanel extends React.Component<SidePanelProps, SidePanelState> {
 
   constructor(props: SidePanelProps) {
     super(props);
-    this.handleResize = throttle(this.handleResize.bind(this), 150);
+    this.handleResize = this.handleResize.bind(this);
```

Comment:
this is the one I am unsure about if we need to keep around...I just deleted it so I can ask the question (also see what tests fail)

Reply:
Strange the function is big. It is kind of inlined in [useTooltip](https://github.com/Workday/canvas-kit/blob/35d47852443e9969eeb77c4f070da29d31912c2c/modules/react/tooltip/lib/useTooltip.tsx#L11-L34)

Reply:
I'm not worried about this.